app/gui2/src/components/GraphEditor/GraphNodes.vue

```diff
@@ -32,13 +32,29 @@ function moveNode(movedId: ExprId, delta: Vec2) {
 function hoverNode(id: ExprId | undefined) {
   if (selection != null) selection.hoveredNode = id
 }
+
+function setEditedNode(id: ExprId | null, cursorPosition: number | null) {
```

Comment:
I believe that this should be a store method. Try to avoid setting the store values directly in components, as it can quickly become pretty hard to track.